client/src/QueueDisplay.js

```diff
@@ -26,7 +26,10 @@ export default function QueueDisplay(props) {
             <div className={i == 0 ? 'userBox firstBox' : 'userBox'} onClick={() => leaveQueue(i)}>
               {item.username}
             </div>
+            <div className='entryTimeContainer'>Starttid:</div>
             <div className='entryTimeContainer'>{formatDate(item.entrytime)}</div>
+            <div className='entryTimeContainer'>Estimert ferdig:</div>
+            <div className='entryTimeContainer'>{formatDate(item.estimatedFinishTime)}</div>
```

Comment:
Potentially a bit redundant to display the date here, when the max limit is 8h anyways? It will typically not go into the next day anyway